modules/ROOT/pages/aurads/importing-data/load-csv.adoc

```diff
@@ -169,19 +155,25 @@ load_people_csv = """
 
 # Create the driver session
 with driver.session() as session:
-    # Load the CSV file
+    # Load the CSV files
+    session.run(load_movies_csv).data()
     session.run(load_people_csv).data()
 ----
 =====
 ====
 
 == Add relationships from CSV files
 
-Similarly to what we have done for nodes, we now create relationships from the `actors.csv` file. In the following examples, `LOAD CSV` is used with:
+Similarly to what we have done for nodes, we now create relationships from the `actors.csv` file.
+In the following example, `LOAD CSV` is used with the `WITH HEADERS` option to access the fields in each `row` by their corresponding column name.
 
-* `WITH HEADERS` to access `row` fields by their corresponding column name;
-* `USING PERIODIC COMMIT` is used to control memory usage by creating a commit every _n_ processed rows rather than after processing the whole file. This is especially useful when loading large files;
-* `FIELDTERMINATOR` is used to explicitly set the field terminator character (although the comma is the default).
+[TIP]
+====
+The default field delimiter for `LOAD CSV` is the comma (`,`).
+Use the link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/load-csv/#load-csv-import-data-from-a-csv-file-with-a-custom-field-delimiter[`FIELDTERMINATOR` option] to set a different delimiter.
+
+If the CSV file is large, use the link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/load-csv/#load-csv-importing-large-amounts-of-data[`CALL IN TRANSACTIONS` clause] to commit a number of rows per transaction instead of the whole file.
```

Comment:
I would link here instead: https://neo4j.com/docs/cypher-manual/current/subqueries/subqueries-in-transactions/#_loading_csv_data

The LOAD CSV page is not in the best state, and I need to improve it.

Reply:
To me the existing link seems more appropriate because it also mentions the reason why you should use that clause, and also the bit about implicit transactions. From there, there is still a link to the page you mentioned. What do you think?

Reply:
Yes, that sounds good 👍